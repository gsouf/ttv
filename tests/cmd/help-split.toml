bin.name = "ttv"
args = "split -h"
