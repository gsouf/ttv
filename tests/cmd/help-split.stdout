ttv-split 
Split dataset into two or more files for test/train/validation sets

USAGE:
    ttv split [OPTIONS] <INPUT>

ARGS:
    <INPUT>    Data to split, optionally gzip compressed. If '-', read from stdin

OPTIONS:
    -c, --chunk-size <CHUNK_SIZE>
            Maximum number of rows per output chunk

    -C, --compressed-output
            Compress output files using gzip

        --csv
            Parse input as CSV. Only needed if rows contain embedded newlines - will impact
            performance.

    -d, --decompress-input
            Decompress input from gzip format

    -h, --help
            Print help information

    -n, --no-header
            Don't treat the first row as a header

    -o, --output-prefix <OUTPUT_PREFIX>
            Output filename prefix. Only used if reading from stdin

    -p, --prop <PROP>
            Specify splits by proportion of rows

    -r, --rows <ROWS>
            Specify splits by number of rows

    -s, --seed <SEED>
            RNG seed, for reproducibility

    -t, --total-rows <TOTAL_ROWS>
            Number of rows in input file. Used for progress when using proportion splits
